lib/context2d.js

```diff
@@ -16,10 +16,6 @@ var canvas = require('./bindings')
   , CanvasPattern = canvas.CanvasPattern
   , ImageData = canvas.ImageData;
 
-var parseCssFont = require('parse-css-font');
```

Comment:
The removal of this should be it's own PR, it also should remove the dependencies from the package.json...